integrations/jira/jira-servicedesk-incidents/README.md

```diff
@@ -0,0 +1,91 @@
+## Overview
+
+<!-- Sensu Integration description; supports markdown -->
+
+The Jira Service Management integration is a Sensu Event Pipeline that generates alerts in the Jira Service Management service.
+
+<!-- Provide a high level overview of the integration contents (e.g. checks, filters, mutators, handlers, assets, etc) -->
+
+This integration provides the following resources:
+
+* `jira-servicedesk-pipeline` [pipeline]
+* `jira-servicedesk-workflow` [workflow]
+* `jira-servicedesk-handler` [handler]
+* `jira_api_key` [secret]
+* `sensu/sensu-jira-handler:2.0.0` [asset]
+
+## Dashboards
+
+<!-- List of supported dashboards w/ screenshots (supports png, jpeg, and gif images; relative paths only; e.g. `![](img/dashboard-1.png)` )-->
+
+This integration is compatible with the Jira Service Management dashboard. The Incidents dashboard will display the Sensu Event namespace, entity, and check details (including check status).
+
+## Setup
+
+<!-- Sensu Integration setup instructions, including Sensu agent configuration and external component configuration -->
+<!-- EXAMPLE: what configuration (if any) is required in a third-party service to enable monitoring? -->
+
+1. **Get a Jira Service Management Integration Key**
+
+   While the on-premise version of Jira uses basic HTTP authentication with only
+   a simple username and password the [cloud version requires the use of an API
+   token to authenticate][10]. To create the API token perform the following steps:
```

Comment:
[10] also not defined.

Probably wanted this from the handler readme:
`[10]: https://confluence.atlassian.com/cloud/deprecation-of-basic-authentication-with-passwords-for-jira-and-confluence-apis-972355348.html`